how_to_build.md

```diff
@@ -5,22 +5,24 @@ It also replicates the build on the Continuous Integration build server and is t
 
 *Don't be put off by the prerequisites!* It only takes a few minutes to set them up and only needs to be done once. If you haven't used [Rake](http://rake.rubyforge.org/ "RAKE -- Ruby Make") before then you're in for a real treat!
 
-You can also build the solution using Visual Studio 2012 or later, but this doesn't provide the same assurances as the Rake build.
+You can also build the solution using Visual Studio 2015 or later, but this doesn't provide the same assurances as the Rake build.
 
 At the time of writing the build is only confirmed to work on Windows using the Microsoft .NET framework.
 
 ## Prerequisites
 
 1. Ensure you have .NET framework 3.5 and 4.0/4.5 installed.
 
+1. Ensure you have Visual Studio 2015 or MSBuild 14 installed.

```

Comment:
This is because MSBuild no longer ships with the framework. It ships with VS releases.